test/integration/builds_int_test.go

```diff
@@ -115,11 +115,11 @@ func (suite *BuildsIntegrationTestSuite) TestBuilds_Download() {
 	cp.ExpectExitCode(0, e2e.RuntimeSourcingTimeoutOpt)
 
 	cp = ts.SpawnWithOpts(
-		e2e.OptArgs("builds", "dl", "16291D17", "."),
+		e2e.OptArgs("builds", "dl", "1f924587", "."),
 	)
-	cp.Expect("Downloaded", e2e.RuntimeSourcingTimeoutOpt)
+	cp.Expect("Downloaded sqlite3", e2e.RuntimeSourcingTimeoutOpt)
```

Comment:
sqlite3 is not tiny :p 

![image](https://github.com/ActiveState/cli/assets/278649/b47b045c-55e1-4da0-9ff0-cad3505e6c4b)

Reply:
It's 2.3 MB! Found a smaller one. In testing this out I also found a different approach rather than aborting the download bar.